testsuites/syncgateway/performance/run_perf_test.py

```diff
@@ -45,42 +45,71 @@ def run_perf_test(number_pullers, number_pushers, use_gateload, gen_gateload_con
-
-        # Build gateload
-        print ">>> Building gateload"
-        ansible_runner = AnsibleRunner()
-        status = ansible_runner.run_ansible_playbook(
-            "build-gateload.yml",
-            extra_vars={},
-        )
-        assert status == 0, "Could not build gateload"
-
-        # Generate gateload config
-        print ">>> Generate gateload configs"
-        if gen_gateload_config:
-            generate_gateload_configs.main(
-                number_pullers,
-                number_pushers,
-                test_run_id,
-                doc_size,
-                runtime_ms,
-                rampup_interval_ms
-            )
-
-        # Start gateload
-        print ">>> Starting gateload with {0} pullers and {1} pushers".format(number_pullers, number_pushers)
-        status = ansible_runner.run_ansible_playbook(
-            "start-gateload.yml",
-            extra_vars={},
-        )
-        assert status == 0, "Could not start gateload"
+    if use_loadgenerator:
+        print "Using Load Generator: {0}".format(use_loadgenerator)
 
+        if use_loadgenerator == '--use-gateload'
+            if int(number_pullers) > 0 and not gen_gateload_config:
+                raise Exception("You specified --num-pullers but did not set --gen-gateload-config")
 
+            # Build gateload
+            print ">>> Building gateload"
+            ansible_runner = AnsibleRunner()
+            status = ansible_runner.run_ansible_playbook(
+                "build-gateload.yml",
+                extra_vars={},
+            )
+            assert status == 0, "Could not build gateload"
+
+            # Generate gateload config
+            print ">>> Generate gateload configs"
+            if gen_gateload_config:
+                generate_gateload_configs.main(
+                    number_pullers,
+                    number_pushers,
+                    test_run_id,
+                    doc_size,
+                    runtime_ms,
+                    rampup_interval_ms
+                )
+
+            # Start gateload
+            print ">>> Starting gateload with {0} pullers and {1} pushers".format(number_pullers, number_pushers)
+            status = ansible_runner.run_ansible_playbook(
+                "start-gateload.yml",
+                extra_vars={},
+            )
+            assert status == 0, "Could not start gateload"
+
+        elif use_loadgenerator == '--use-sgload':
+            # Build sgload
+            print ">>> Building sgload"
+            ansible_runner = AnsibleRunner()
+            status = ansible_runner.run_ansible_playbook(
+                "build-sgload.yml",
+                extra_vars={},
+            )
+            assert status == 0, "Could not build sgload"
+
+            # Generate sgload config
+            print ">>> Generate sgload configs"
+            if gen_sgload_config:
+                generate_sgload_configs.main(
+                    number_pullers,
```

Comment:
Looks like neither `gen_sgload_config` or `generate_sgload_configs` are defined anywhere.  `sgload` doesn't currently use a config file, it takes all args on the command line